renovate-schema.json

```diff
@@ -36,7 +36,7 @@
     "forceCli": {
       "description": "Whether CLI configuration options should be moved to the `force` config section",
       "type": "boolean",
-      "default": false
+      "default": true
```

Comment:
Nope :)

Reply:
That's has happened because renovate schema was different, I just ran the script to update the schema. I guess the definitions have changed in master, I had to do it for all my PRs today @rarkins